samples/13.core-bot/src/main/java/com/microsoft/bot/sample/core/DialogBot.java

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.bot.sample.core;
+
+import com.microsoft.bot.builder.*;
+import com.microsoft.bot.dialogs.Dialog;
+
+import java.util.concurrent.CompletableFuture;
+
+// This IBot implementation can run any type of Dialog. The use of type parameterization is to allows multiple different bots
+// to be run at different endpoints within the same project...
```

Comment:
Are you missing the comment of L16 and L17? See [C#](https://github.com/microsoft/BotBuilder-Samples/blob/main/samples/csharp_dotnetcore/13.core-bot/Bots/DialogBot.cs#L16)

Reply:
You say I add those two lines and leave it at that? Could be 🤔

Reply:
That's better than the `...`, right?

Reply:
I was not going to leave the comment like that, you seen my post on the thread?

Reply:
I'll check and update this comment asap

Reply:
I think that we can follow the behavior of [EchoBot](https://github.com/microsoft/botbuilder-java/blob/main/samples/47.inspection/src/main/java/com/microsoft/bot/sample/inspection/EchoBot.java#L15).

C# and Java commentaries are not the same, but the Java describes correctly the content of the EchoBot as we need in this bot